WordPress/Classes/ViewRelated/Me/Account Settings/AccountSettingsViewController.swift

```diff
@@ -274,45 +274,55 @@ private class AccountSettingsController: SettingsController {
-        case "atomic-site":
-            return NSLocalizedString("This user account cannot be closed while it has active atomic sites.",
-                                     comment: "Error message displayed when unable to close user account due to having active atomic site.")
-        case "chargebacked-site":
-            return NSLocalizedString("This user account cannot be closed if there are unresolved chargebacks.",
-                                     comment: "Error message displayed when unable to close user account due to unresolved chargebacks.")
-        case "active-subscriptions":
-            return NSLocalizedString("This user account cannot be closed while it has active subscriptions.",
-                                     comment: "Error message displayed when unable to close user account due to having active subscriptions.")
-        case "active-memberships":
-            return NSLocalizedString("This user account cannot be closed while it has active purchases.",
-                                     comment: "Error message displayed when unable to close user account due to having active purchases.")
+        switch error {
+        case .atomicSite:
+            let title = NSLocalizedString("Contact Support",
+                                          comment: "Title for a button displayed when unable to close user account due to having atomic site.")
+            alert.addActionWithTitle(title, style: .default, handler: contactSupportAction)
+            let cancelAction = NSLocalizedString("Cancel", comment: "Alert dismissal title")
+            alert.addCancelActionWithTitle(cancelAction)
         default:
-            return NSLocalizedString("An error occured while closing account.",
-                                     comment: "Default error message displayed when unable to close user account.")
+            let okAction = NSLocalizedString("OK", comment: "Alert dismissal title")
+            alert.addDefaultActionWithTitle(okAction)
         }
-    }
 
-    private func showErrorAlert(message: String) {
-        let title = NSLocalizedString("Error", comment: "General error title")
-        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
-        let okAction = NSLocalizedString("OK", comment: "Alert dismissal title")
-        alert.addDefaultActionWithTitle(okAction, handler: nil)
         alert.presentFromRootViewController()
     }
 
+    private var contactSupportAction: ((UIAlertAction) -> Void) {
+        return { action in
+            if ZendeskUtils.zendeskEnabled {
+                guard let leafViewController = UIApplication.shared.leafViewController else {
+                    return
+                }
+                ZendeskUtils.sharedInstance.showNewRequestIfPossible(from: leafViewController, with: nil) { [weak self] identityUpdated in
```

Comment:
@nikola-milicevic (cc @leandroalonso)
Let's set a support source tag for this action. You can add a new `closeAccount` support source tag
[here](https://github.com/wordpress-mobile/WordPress-iOS/blob/a3eae1d75066b28c215180cafd12e4947c74e556/WordPress/Classes/ViewRelated/NUX/WordPressSupportSourceTag%2BHelpers.swift)

This will add a tag to the Zendesk ticket, which will help our HEs sort the tickets! 😄

Reply:
Sounds good! 👍